pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: bazel_worker
-version: 0.1.5
+version: 0.1.6
```

Comment:
You can leave this at 0.1.5. I didn't publish 0.1.5 yet or roll it into Google. (I didn't even notice you'd merged my PR.)

Reply:
I got a bit overzealous and already published lol, but it doesn't really matter since these aren't breaking versions.